client/test/bpmn-io-modelers/dmn/DmnModelerSpec.js

```diff
@@ -0,0 +1,418 @@
+    // then
+    expect(modeler).to.exist;
+  });
+
+
+  describe('events', function() {
+
+    let modeler;
+
+    beforeEach(async function() {
+      modeler = await createModeler({
+        container: modelerContainer
+      });
+    });
+
+
+    it('view.contentChanged', function() {
+
+      // given
+      const eventBus = modeler.getActiveViewer().get('eventBus');
+
+      const spy = sinon.spy();
+
+      modeler.on('view.contentChanged', spy);
+
+      // when
+      eventBus.fire('commandStack.changed');
+
+      // then
+      expect(spy).to.have.been.called;
+    });
+
+
+    it('view.selectionChanged', function() {
+
+      // given
+      const eventBus = modeler.getActiveViewer().get('eventBus');
+
+      const spy = sinon.spy();
+
+      modeler.on('view.selectionChanged', spy);
+
+      // when
+      eventBus.fire('selection.changed', { oldSelection: [], newSelection: [] });
+
+      // then
+      expect(spy).to.have.been.called;
+    });
+
+
+    it('view.directEditingChanged', function() {
+
+      // given
+      const eventBus = modeler.getActiveViewer().get('eventBus');
+
+      const spy = sinon.spy();
+
+      modeler.on('view.directEditingChanged', spy);
+
+      // when
+      eventBus.fire('directEditing.activate');
+
+      // then
+      expect(spy).to.have.been.called;
+    });
+
+
+    it('error', function() {
+
+      // given
+      const eventBus = modeler.getActiveViewer().get('eventBus');
+
+      const spy = sinon.spy();
+
+      modeler.on('error', spy);
+
+      // when
+      eventBus.fire('error');
+
+      // then
+      expect(spy).to.have.been.called;
+    });
+
+
+    it('propertiesPanel.focusin', function() {
+
+      // given
+      const eventBus = modeler.getActiveViewer().get('eventBus');
+
+      const spy = sinon.spy();
+
+      modeler.on('propertiesPanel.focusin', spy);
+
+      // when
+      eventBus.fire('propertiesPanel.focusin');
+
+      // then
+      expect(spy).to.have.been.called;
+    });
+
+
+    it('propertiesPanel.focusout', function() {
+
+      // given
+      const eventBus = modeler.getActiveViewer().get('eventBus');
+
+      const spy = sinon.spy();
+
+      modeler.on('propertiesPanel.focusout', spy);
+
+      // when
+      eventBus.fire('propertiesPanel.focusout');
+
+      // then
+      expect(spy).to.have.been.called;
+    });
+
+  });
+
+
+  it('#getStackIdx', async function() {
+
+    // when
+    const modeler = await createModeler({
+      container: modelerContainer
+    });
+
+    // then
+    expect(modeler.getStackIdx()).to.equal(-1);
+  });
+
+
+  describe('overview', function() {
+
+    let modeler;
+
+    beforeEach(async function() {
+      modeler = await createModeler({
+        container: modelerContainer
+      });
+
+      modeler.attachOverviewTo(overviewContainer);
+    });
+
+
+    it('should have overview', function() {
+
+      // then
+      expect(modeler._overview).to.exist;
+      expect(modeler._overview instanceof DrdViewer).to.be.true;

```

Comment:
Let's use [instanceof chai method](https://www.chaijs.com/api/bdd/#method_instanceof) for that

```suggestion
      expect(modeler._overview).to.be.instanceof(DrdViewer);
```

Thus, we can get helpful error message instead of _expected false to be true_.